src/ProjectTemplates/test/Templates.Mvc.Tests/Templates.Mvc.Tests.csproj

```diff
@@ -64,4 +64,13 @@
 
   <!-- Shared testing infrastructure for running E2E tests -->
   <Import Condition="'$(SkipTestBuild)' != 'true'" Project="..\..\TestInfrastructure\PrepareForTest.targets" />
+
+  <!-- See https://github.com/dotnet/aspnetcore/issues/55695 -->
```

Comment:
If we just want to skip these on Helix, set `<BuildHelixPayload>false</BuildHelixPayload>`